OpenTabletDriver.Plugin/Output/AbsoluteOutputMode.cs

```diff
@@ -153,7 +153,7 @@ protected override void OnOutput(IDeviceReport report)
             if (report is ITiltReport tiltReport && Pointer is ITiltHandler tiltHandler)
                 tiltHandler.SetTilt(tiltReport.Tilt);
             if (report is ITabletReport tabletReport && Pointer is IPressureHandler pressureHandler
-                && Tablet.Properties.Specifications.Pen != null)
+                && Tablet != null && Tablet.Properties.Specifications.Pen != null)
```

Comment:
```suggestion
                && Tablet?.Properties?.Specifications?.Pen != null)
```
might as well do this

Reply:
`Tablet?.Overkill.Overkill.Pen != null`

Reply:
Well, check warnings will continue if you don't do it this way. But approving anyway since the remaining check warnings will be fixed in #3691.

Reply:
What warning (there are none caused by me), also they don't happen locally, i have also noticed found many issues with C# & github's CI yesterday alone.
Btw you shouldn't make projects Nullabke if they are not built with it in mind.